src/ClientData/include/ClientData/ThreadTrackPaneData.h

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CLIENT_DATA_THREAD_TRACK_PANE_DATA_H_
+#define CLIENT_DATA_THREAD_TRACK_PANE_DATA_H_
+
+#include <absl/synchronization/mutex.h>
+
+#include <vector>
+
+#include "Containers/ScopeTree.h"
+#include "TrackPaneData.h"
+
+namespace orbit_client_data {
+
+class ThreadTrackPaneData final {
```

Comment:
I'm calling this class ThreadTrackPaneData because we've already had a TrackPaneData class. So, I will answer why it's TrackPaneData.

A few weeks ago it was being called TrackData because it was going to be the data from Tracks but actually it turned to be only the Timer's data, so it's only used in TimerTracks. In that context, TrackData is confusing so I supposed that Dimitry renamed the class for that reason.

I would like to propose a different name, because the concept of TimerPane was proposed in a design doc, where Timers belong to a Pane, and a Track is composed by several Panes. As we can have a "SamplePane", where we have samples instead of Timers, the more generic name "EventPane" came in a following doc. In this context, I would call TimerPaneData to the current TrackPaneData and ThreadTimerPaneData to ThreadTrackPaneData if both of you agreed.

Reply:
Added a description.

Reply:
Yeah... I'm really not sure about the naming. Having this tied to `ThreadTrack` seems wrong. What if a GpuTrack also wants ScopeTree functionality? Should it own a `ThreadTrackPaneData` object?

I'd suggest going for `TimerDataProvider` and `ScopeTreeTimerDataProvider` instead, or maybe even simply `TimerData` and `ScopeTreeTimerData`. I think it would be easier to understand what those classes are. I don't think "Pane" adds anything, so I'd remove it.

Reply:
WDYT @karupayun and @dimitry- ?

Reply:
Yes, I agree

Reply:
sgtm

Reply:
It's renamed to ScopeTreeTimerData. The Provider will come soon and will manage all the data. And in a following PR I will rename TrackPaneData to TimerData/Provider

Reply:
Sounds good, thanks for the change.